src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -69,30 +69,38 @@ const handleNextStepEvent = (wizardEventName: string, nextStepName: string) => {
   event(`firstMile.${wizardEventName}.nextSteps.${nextStepName}.clicked`)
 }
 
+const marginTopStyle = {marginTop: '80px'}
+
 export const Finish = (props: OwnProps) => {
+  const {
+    finishStepCompleted,
+    wizardEventName,
+    finalFeedback,
+    markStepAsCompleted,
+    setFinalFeedback,
+  } = props
+
   useEffect(() => {
     // if the finish step was opened during the session,
     // this check prevents from multiple logging of finish events
     // in case user navigates back and forth
-    if (!props.finishStepCompleted) {
-      event(`firstMile.${props.wizardEventName}.finished`)
-      props.markStepAsCompleted()
+    if (!finishStepCompleted) {
+      event(`firstMile.${wizardEventName}.finished`)
+      markStepAsCompleted()
       fireConfetti()
     }
-  }, [])
+  }, [finishStepCompleted, markStepAsCompleted, wizardEventName])
```

Comment:
This is kind of an interesting case that it's worth discussing, so let's pair up on it briefly. I believe this is the right result, but it's worth walking through the code a bit to talk about why it's all good to add `finishStepCompleted` here.